### ISSUE TITLE
cannot browse with screen reader between plugins in NPP (plugins -> plugin administrator)

### ISSUE THREAD

**Issue by gregjozk** [notepad-plus-plus/nppPluginList#444]
Hello,

don't know if my issue should be posted here or on main npp github...
anyway here are STR:
- start NVDA or other screen reader on Windows
- start NPP (I have 8.3.3 32-bit)
- go to plugins
- open plugin admin

if I have not used advanced moving commands, I did not see any plugin in all 3 tabs like available, update or updated.

it would be greatly appretiated, if screen reader's focus could land on plugin list only with pressing "tab key" like for landing on a search field, buttons "previous/next" etc.

thanks.

Jožef

**Comment by chcg** [comment]
@gregjozk This should be added to the N++ issues as it needs to be fixed by the application itself and not here.
